<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for odoo-addon-purchase-request-exception</title>
  </head>
  <body>
    <h1>Links for odoo-addon-purchase-request-exception</h1>
    <a href="https://files.pythonhosted.org/packages/c0/91/b7a7966bccba18e405b356e8a959bd88a2687175df3b261d91fc307db30f/odoo_addon_purchase_request_exception-15.0.1.0.0.2-py3-none-any.whl#sha256=5e5e23e7cae27711da41c7995534598840aaf296f227ea6985c326a5c6f35fd8" data-requires-python="&gt;=3.8" >odoo_addon_purchase_request_exception-15.0.1.0.0.2-py3-none-any.whl</a><br />
    <a href="https://files.pythonhosted.org/packages/7f/ad/72226e598c94dcb23037cf1a56c9f930aba6e5fd208c24f5fed4cf638d42/odoo_addon_purchase_request_exception-15.0.1.0.0.4-py3-none-any.whl#sha256=6ce03f644b297c54e728f0ee9ce99606d8ed33f9e2d4a5181d8793537bb8ede4" data-requires-python="&gt;=3.8" data-dist-info-metadata="sha256=4e911c60cbe813e85f15610041573d40337f6bf16f952e3762fdd5dea3807151" data-core-metadata="sha256=4e911c60cbe813e85f15610041573d40337f6bf16f952e3762fdd5dea3807151">odoo_addon_purchase_request_exception-15.0.1.0.0.4-py3-none-any.whl</a><br />
    <a href="https://files.pythonhosted.org/packages/4c/4f/a1e8f57ba9fad71738898245fdde6f994fa446aa9c7853831a2f85f94ab5/odoo_addon_purchase_request_exception-15.0.1.0.0.5-py3-none-any.whl#sha256=8a42dacc8cd63dc5fd1a8bfa0ff28d0a1ab66bb89ccd0d44167b53fd81b49530" data-requires-python="&gt;=3.8" data-dist-info-metadata="sha256=f3f55bff7beaaad9524ac44f9415f7f6d18bc3826125a743f40a3c4cb20bf7f3" data-core-metadata="sha256=f3f55bff7beaaad9524ac44f9415f7f6d18bc3826125a743f40a3c4cb20bf7f3">odoo_addon_purchase_request_exception-15.0.1.0.0.5-py3-none-any.whl</a><br />
    </body>
</html>
<!--SERIAL 19587747-->